src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,14 @@
+export const namesHints = {
+  name: 'Unique identifier that may be configured to point to accounts, contracts, oracles, and NFTs.',
+  nameInAuction: 'Name that points to accounts, contracts, oracles, NFTs and more. Names shorter than 13 characters need to be auctioned.',
+  expires: 'Estimated date, time, and block height when the name will be detached from the account.',
+  buyer: 'Address of the account which paid for the Name.',
+  owner: 'Owner of the Name.',
+  pointsTo: 'Address that Name points to.',
+  highestBid: 'Account of the highest bidder and the price of the highest bid.',
+  bidCount: 'Amount of bids in the auction.',
+  ends: 'Estimated date, time, and block height when the auction ends.',
```

Comment:
To me it looks like block height is also estimated. I would propose the following:
`Block height and estimated date and time when the auction ends.`